src/components/AppLayout/Header/components/ProviderDetails/HidePairingModule.tsx

```diff
@@ -0,0 +1,6 @@
+import 'src/components/AppLayout/Header/components/ProviderDetails/hidePairingModule.css'
```

Comment:
Shouldn't this dynamically inject when (un-)mounting?

Reply:
Mobile Pairing is enabled on every network now so we don't need to remove the css anymore on unmount

Reply:
Good point! We can change it back to how it was before in case a network doesn't support it but including it in the `Header` to solve this issue